modules/bookshelf.js

```diff
@@ -0,0 +1,66 @@
+const storageKey = 'awesomeBooks';
+const timeNow = new Date();
+
+export default class BookShelf {
+  constructor() {
+    this.arrBooks = [];
+  }
+
+  addBook(newTitle, newAuthor) {
+    const newBook = {
+      id: timeNow.getTime(),
+      title: newTitle,
+      author: newAuthor,
+    };
+
+    this.arrBooks.push(newBook);
+  }
+
+  removeBook(id) {
+    this.arrBooks = this.arrBooks.filter((book) => book.id !== id);
+  }
+
+  saveDataToLocalStorage() {
+    localStorage.setItem(storageKey, JSON.stringify(this.arrBooks));
+  }
+
+  showBooks() {
+    function addElement(elementType, parent, className) {
+      const element = document.createElement(elementType);
+      element.classList.add(className);
+      parent.appendChild(element);
+      return element;
+    }
+
+    const bookList = document.querySelector('.book-list');
+    this.arrBooks.forEach((book) => {
+      const bookItem = addElement('div', bookList, 'book-item');
+      const bookTitle = addElement('div', bookItem, 'book-title');
+      bookTitle.innerHTML = `" ${book.title} "    by ${book.author}`;
+
+      const bookRemoveButton = addElement(
+        'button',
+        bookItem,
+        'book-remove-button',
+      );
+      bookRemoveButton.innerHTML = 'Remove';
+
+      bookRemoveButton.addEventListener('click', () => {
+        this.removeBook(book.id);
+        this.saveDataToLocalStorage();
+        window.location.reload();
+      });
+    });
+  }
+
+  loadDataFromLocalStorage() {
+    const dataLoaded = JSON.parse(localStorage.getItem(storageKey));
+    if (dataLoaded == null) {
+      this.arrBooks = [];
+    } else {
+      this.arrBooks = dataLoaded;
+    }
+
+    this.showBooks();
+  }
+}
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/213353465-f16ac835-64e5-4538-966b-ba546095e745.png)

- [ ] Your methods in the class are not arrow methods :fearful: Kindly convert all these class methods to arrow methods as I did in the image above :arrow_up: to follow ES6 practices :+1: